st2common/bin/st2-track-result

```diff
@@ -44,73 +44,5 @@ def setup():
     script_setup.setup(config, register_mq_exchanges=False)
 
 
-def add_result_tracker(exec_id):
```

Comment:
Can we keep the st2 result tracker and minimize changes to it?  st2 result tracker is a generic solution to pull results from a remote runner instead of having the runner push result into st2.  It is not specific to Mistral. Although Mistral is the only one that uses it.

Reply:
> Can we keep the st2 result tracker and minimize changes to it? st2 result tracker is a generic solution to pull results from a remote runner instead of having the runner push result into st2. It is not specific to Mistral. Although Mistral is the only one that uses it.

Hi @m4dcoder I've put back the st2-track-result back to where it was, but with an additional comment to explain why it's there, and to update when other runners support the result tracker.